contracts/delegate.md

```diff
@@ -8,39 +8,40 @@ Create a new `Delegate` contract.
 
 ```java
 constructor(
-  ISwap _swapContract,
-  address _delegateContractOwner,
-  address _delegateTradeWallet
+  ISwap delegateSwap,
+  IIndexer delegateIndexer,
+  address delegateContractOwner,
+  address delegateTradeWallet
 ) public
 ```
 
 | Param                    | Type      | Description                                             |
 | :----------------------- | :-------- | :------------------------------------------------------ |
-| `_swapContract`          | `ISwap`   | Instance of the swap contract used to settle trades.    |
-| `_delegateContractOwner` | `address` | Address of the owner of the sender for rule management. |
-| `_delegateTradeWallet`   | `address` | Address of the wallet that holds funds to be traded.    |
+| `swapContract`          | `ISwap`   | Instance of the swap contract used to settle trades.    |
+| `delegateContractOwner`  | `address` | Address of the owner of the sender for rule management. |
+| `delegateTradeWallet`    | `address` | Address of the wallet that holds funds to be traded.    |
 
 ## `setRule`
 
 Set a trading rule on the delegate. Delegate assumes the role of sender.
```

Comment:
Could add an explanation of `1 senderToken = priceCoef * 10^(-priceExp) * signerToken` to fully explain the numbers in the rule